### ISSUE TITLE
Update Mohist PSA to reflect SpongeForge status

### ISSUE THREAD

**Issue by imDaniX** [EssentialsX/Website#78]
SpongeForge is available to download on 1.16.5

**Comment by Outfit8TSB** [comment]
Yeah, i created a [pull request](https://github.com/EssentialsX/website/pull/101) #101 that closes this issue but it was closed & locked for Not Ready. Maybe i will only slight edit. However one thing that they missed was Mohist Doesnt have Native english speakers in their team(seriously, just not looking at Warning Message Errors very very considerly was Racist). EssentialsX Might Reconsider This: 1. Contribute to Forge-Bukkit Hybrid translations or, 2. Make a sponge portion of EssentialsX for spongeforge 1.20.6~1.21, plus, Where is FTB Essentials Neoforge? I Mean Like What The heck

**Comment by mdcfe** [comment]
To quote:  
> All hybrid server software is unsupported, and Mohist has eroded trust as a project. We will update our information about hybrid software and alternatives when we are ready as a team.

We're aware that the alternatives have changed and moved on. We intend to update support information in the future, hence the reason this issue is still open. In the meantime, we trust that users are capable of researching alternatives on their own. There is no need to bump this issue, nor to create clearly-opinionated PRs that contradict our support policy.
